### PR TITLE
fix(liveslots): update stale comment about addRecognizableValue

### LINKED ISSUE #6778: stale comment in addRecognizableValue ?
While accidentally rediscovering #4834, I came across a [comment](https://github.com/Agoric/agoric-sdk/blob/229708bcc66afa53dbab929ce1826787942179f7/packages/SwingSet/src/liveslots/virtualReferences.js#L460-L474) in [`addRecognizableValue()`](https://github.com/Agoric/agoric-sdk/blob/229708bcc66afa53dbab929ce1826787942179f7/packages/SwingSet/src/liveslots/virtualReferences.js#L495) which might be stale:

```js
  /**
   * A map from vrefs (those which are recognizable by (i.e., used as keys in)
   * VOM aware collections) to sets of recognizers (the collections for which
   * they are respectively used as keys).  These vrefs correspond to either
   * imported Presences or virtual objects (Remotables do not participate in
   * this as they are not keyed by vref but by the actual Remotable objects
   * themselves). We add to a vref's recognizer set whenever we use a Presence
   * or virtual object as a key into a weak store instance or an instance of
   * VirtualObjectAwareWeakMap or VirtualObjectAwareWeakSet.  We remove it
   * whenever that key (or the whole collection containing it) is deleted.
   *
   * A recognizer is one of:
   *   Map - the map contained within a VirtualObjectAwareWeakMap to point to its vref-keyed entries.
   *   Set - the set contained within a VirtualObjectAwareWeakSet to point to its vref-keyed entries.
   *   deleter - a function within a WeakStore that can be called to remove an entry from that store.
   */
  /**
   * @typedef { Map<string, *> | Set<string> } Recognizer
   */
  /** @type {Map<string, Set<Recognizer>>} */

  const vrefRecognizers = new Map();
```

Before #4834 was fixed by #5059 (by moving the recognizer edges into the vatstore), we used this `vrefRecognizers` to track these edges. The key of this Map is the baseref of a virtual object, and the value describes the (weak) collection that can recognize it (by virtue of the vobj being used as a key in a WeakMap, or as a member of a WeakSet).

Originally, this was only used for `VirtualObjectAwareWeakMap/Set`, which are not themselves virtual, so they consume RAM, and therefore any auxilliary tables (like these reference trackers) are also allowed to consume RAM: it doesn't violate our memory goals. So the values in `vrefRecognizers` were either the underlying (real) Map or Set, from which we should delete the vref if/when the virtual object went away.

Then we added virtual collections, some of which are weak, and these had the same need for tracking recognizability. At that time, I think `vrefRecognizers` grew to include a third type of value, which was a `deleter` function owned by a specific (virtual, weak) collection. But that violated our memory goals (holding tracking data in RAM for a purely virtual object), so #4834 was about fixing it. #5059 moved that table into the vatstore.

But I'm thinking that the comment didn't get updated. The typedef did, but the prose still talks about a `deleter`, and the third sentence says "as a key into a weak store instance" which I think is no longer a real case.

Anyways, I got tripped up by the comment while working on something else, and thought I'd discovered a memory-consumption bug, when in fact the bug was fixed 9 months ago.

### PR DESCRIPTION
**Pull request by warner**
closes #6778

### CODE REVIEW

File: packages/SwingSet/src/liveslots/virtualReferences.js [hunk 1]
```diff
@@ -460,20 +460,20 @@ export function makeVirtualReferenceManager(
...
-   *   deleter - a function within a WeakMapStore that can be called to remove an entry from that store.
+   * This 'vrefRecognizers' is a Map from those vrefs to the set of
+   * recognizing weak collections, for virtual keys and non-virtual
+   * collections. Specifically, the vrefs correspond to imported
+   * Presences or virtual-object Representatives (Remotables do not
+   * participate: they are keyed by the actual Remotable object, not
+   * its vref). The collections are either a VirtualObjectAwareWeakMap
+   * or a VirtualObjectAwareWeakSet. We remove the entry when the key
+   * is removed from the collection, and when the entire collection is
+   * deleted.
```
Comment (FUDCo) [inline packages/SwingSet/src/liveslots/virtualReferences.js:476]: I recall updating this comment already in a recent PR, so I'm concerned something is out of alignment merge-wise.

Comment (warner) [reply packages/SwingSet/src/liveslots/virtualReferences.js:476]: hm, I don't see any recent edits to that file on trunk.. is it in a PR still in review?

Comment (warner) [reply packages/SwingSet/src/liveslots/virtualReferences.js:476]: we couldn't find the edit, so we'll land this and then merge in the other changes if/when we find them

**Review by FUDCo — APPROVED** [review]
Royt.

**Pull request merged**